bin/merge

```diff
@@ -49,6 +49,8 @@ if [[ "$merged" == "false" ]]; then
     cd $directory
     git checkout -b $tmp_branch $base_sha
     git pull --ff-only $pr_clone $pr_ref
+    git checkout $base_ref
+    git merge --ff-only -m "$(printf "Merge branch '%s'\n\nCloses #%s." "$pr_ref" "$pull_request")" $tmp_branch
```

Comment:
Requires fast-forward merge (`--ff-only`) - this means that if there are going to be conflicts when merging in master, you will need to merge master in manually (and let jenkins run the tests) before running the merge script.

Note that it is possible to merge something into master with broken tests. But best practice currently is only merging to master after jenkins builds are green.